ansible/library/bgp_facts.py

```diff
@@ -143,6 +148,11 @@ def parse_neighbors(self):
                         if regex_peer_group.match(line): neighbor['peer group'] = regex_peer_group.match(line).group(1)
                         if regex_subnet.match(line): neighbor['subnet'] = regex_subnet.match(line).group(1)
 
+                        if regex_cap_gr.match(line): capabilities['graceful restart'] = regex_cap_gr.match(line).group(1).lower()
+                        if regex_cap_gr_peer_restart_time.match(line): capabilities['peer restart timer'] = int(regex_cap_gr_peer_restart_time.match(line).group(1))
+                        if regex_cap_gr_peer_af_ip4.match(line): capabilities['peer af ipv4 unicast'] = regex_cap_gr_peer_af_ip4.match(line).group(1).lower()
+                        if regex_cap_gr_peer_af_ip6.match(line): capabilities['peer af ipv6 unicast'] = regex_cap_gr_peer_af_ip6.match(line).group(1).lower()
+
```

Comment:
can you please make a loop here?

Reply:
Hi Pavel, I don't fully understand your comments. 
IMO, they use different regular expressions to parse and extract different information and don't share any common thing. in this sense, they can hardly be written by a loop.
if you have other ideas could you elaborate them in detail?
thank you

Reply:
ok. Nevermind.